app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,68 @@
+class ArticlesController < ApplicationController
+  load_and_authorize_resource
+  before_action :set_article, only: %i[ show edit update destroy ]
+
+  def index
+    @articles = Article.all
+  end
+
+  def show
+    @articles = Article.find(params[:id])
+  end
+
+  def new
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
+
+    # El metodo current_user, lo provee device a todos los controladores y vistas
+    # para acceder a el usuario logueado
+    @article.owner = current_user
+    respond_to do |format|
+      if @article.save
+        format.html { redirect_to @article, notice: "Article created." }
+        format.json { render :show, status: :created, location: @article }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @article.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def edit
+    @article = Article.find(params[:id])
+  end
+
+  def update
+    respond_to do |format|
+      if @article.update(article_params)
+        format.html { redirect_to @article, notice: "Article updated." }
+        format.json { render :show, status: :ok, location: @article }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @article.errors, status: :unprocessable_entity }
```

Comment:
@sansuaza why is the **format.json** required here?

Reply:
because the messages error and the status need to be packaged to be showed to the client

Reply:
@sansuaza I think you are not performing a json request to the controller .. said this the JSON response format is not really needed